python/torch_mlir/_torch_mlir_custom_op_example/CMakeLists.txt

```diff
@@ -1,5 +1,5 @@
 # Setup PyTorch
-list(APPEND CMAKE_MODULE_PATH "${CMAKE_CURRENT_SOURCE_DIR}/../dialects/torch/importer/jit_ir/cmake/modules")
+list(APPEND CMAKE_MODULE_PATH "${CMAKE_CURRENT_SOURCE_DIR}/../cmake/modules")
 include(TorchMLIRPyTorch)
 TorchMLIRProbeForPyTorchInstall()
```

Comment:
you want to check if it is a LOCAL Pytorch install

Reply:
I changed this PR to just enable custom op support, will modify this in the following PR which will have this example working